src/conv_transpose2d.js

```diff
@@ -115,8 +116,8 @@ export function convTranspose2d(
             ih += realStrideHeight, ++oh) {
             for (let iw = -realBeginningPaddingWidth, ow = 0; ow < outputWidth;
               iw += realStrideWidth, ++ow) {
-              const effectiveOutputChannel = oc + g * outputChannelsPerGroup;
-              const outputLocation = [ib, effectiveOutputChannel, oh, ow];
```

Comment:
Keep these two lines and use `oc` directly to access filter value at line 143?
```js
const filterValue = filter.getValueByLocation(
    [oc, ic, filterHeight - realKh - 1, filterWidth - realKw - 1]);
```

Reply:
Done.